Mathlib/Algebra/Lie/Submodule.lean

```diff
@@ -235,21 +236,25 @@ theorem lie_mem_left (I : LieIdeal R L) (x y : L) (h : x ∈ I) : ⁅x, y⁆ ∈
 #align lie_mem_left lie_mem_left
 
 /-- An ideal of a Lie algebra is a Lie subalgebra. -/
+-- Porting note: added `@[coe]` to fix `norm_cast` issues, but this causes bad pretty-printing.
```

Comment:
Could you please elaborate? What kind of `norm_cast` issues? What kind of bad pretty-printing? Is this a known issue?

Reply:
@Ruben-VandeVelde

Reply:
`(I : LieSubalgebra R L)` becomes `(↑R L I)`